### PR TITLE
Initial code commit

### PR DESCRIPTION
**Pull request by RomainAit**

### CODE REVIEW

File: test_functions/volcano_struct.m [hunk 1]
```diff
@@ -0,0 +1,49 @@
+function [prm, f, s_trnsf] = volcano_struct(pool)
```
Comment (jbect) [inline test_functions/volcano_struct.m]: @RomainAit can we remove the input argument pool? It seems unused.

**Comment by jbect** [comment]
@RomainAit I have squashed everything into one single commit.  For me this PR is almost ready for merging.

Before we merge you may want to
- [x] Remove the unused `pool` argument (see https://github.com/stk-kriging/contrib-qsi/pull/1#discussion_r1415841576)
- [x] Move the functions in `tools` somewhere in `methods`

File: test_functions/volcano_struct.m (hunk 1 above)
Comment (RomainAit) [reply test_functions/volcano_struct.m]: Done.

**Comment by jbect** [comment]
@RomainAit can I squash & rebase+merge this PR https://github.com/stk-kriging/contrib-qsi/pull/1, or do you need more time to make other changes?

(Also, please check https://github.com/stk-kriging/contrib-qsi/pull/1/commits/2f778060004deaa6b6d564b55955b473e74d48ca before you give me the green light)

**Comment by RomainAit** [comment]
Ok for squash and merge !

**Pull request merged**